### ISSUE TITLE
Subclass[COMMON-LISP:SIMPLE-BIT-VECTOR] must implement method[aset_unsafe] 

### ISSUE THREAD

**Issue by kpoeck** [drmeister/clasp#247]
```
> (lisp-implementation-version)

"cclasp-boehm-0.4.0-2-g6b956b1"

> (make-array 10 :element-type 'bit :initial-element 1)
# *0000000000

> (setf (sbit \* 5) 0)

Condition of type: SIMPLE-PROGRAM-ERROR
File(../../include/clasp/core/lispVector.h) lineNumber(79): Subclass[COMMON-LISP:SIMPLE-BIT-VECTOR] must implement method[aset_unsafe] 

Available restarts:
(use :r1 to invoke restart 1)
```
same error with bit instead of sbit

**Comment by Shinmera** [comment]
Can confirm on e6bb007e1e08efd009471cff666436c95945c69d

**Comment by Shinmera** [comment]
Still there on dcc6f10e04e998f628a90922797eeca6dd8b68f9

**Comment by drmeister** [comment]
Fixed and regression tests added

**Comment by Shinmera** [comment]
Confirmed fixed on 12865a162269a6e1b3e6c7392e09b841b69f2cea
